# ImuDebug.msg
Header header
sensor_msgs/Imu imu
int32 counter